### ISSUE TITLE
seccomp section in compose file expects different paths depending on the host OS

### ISSUE THREAD

**Issue by mcartoixa** [microsoft/vscode-remote-release#7271]
- VSCode Version: 1.71.2
- Local OS Version: Windows 10, Ubuntu 22.04, MacOS 12.6

Steps to Reproduce:

I am setting up a containerized development environment based on a Ruby image (on Debian). We use Yarn and our integration tests require Google Chrome so we need to set up [security options](https://docs.docker.com/compose/compose-file/#security_opt) in our docker-compose.yml file, like [this one for chrome](https://github.com/jessfraz/dotfiles/blob/master/etc/docker/seccomp/chrome.json). The corresponding section in the file looks like:
```yml
    security_opt:
      - seccomp:chrome.json
      - seccomp:yarn.json
```
The settings above will work fine on Windows 10 and MacOS 12.6 but will fail on Ubuntu 22.04 with the following logs:
```
[2022-09-30T19:45:38.071Z] Start: Run: docker-compose -f /home/mcartoixa/Développement/air-indemnite.com/airindemnite_client/.devcontainer/docker-compose.yml config
[2022-09-30T19:45:38.257Z] Stop (186 ms): Run: docker-compose -f /home/mcartoixa/Développement/air-indemnite.com/airindemnite_client/.devcontainer/docker-compose.yml config
[2022-09-30T19:45:38.257Z] 
[2022-09-30T19:45:38.257Z] Error reading seccomp profile: [Errno 2] No such file or directory: 'chrome.json'

[2022-09-30T19:45:38.257Z] Exit code 1
[2022-09-30T19:45:38.258Z] Error: Command failed: docker-compose -f /home/mcartoixa/Développement/air-indemnite.com/airindemnite_client/.devcontainer/docker-compose.yml config
[2022-09-30T19:45:38.258Z]     at gc (/home/mcartoixa/.vscode/extensions/ms-vscode-remote.remote-containers-0.251.0/dist/spec-node/devContainersSpecCLI.js:248:907)
[2022-09-30T19:45:38.258Z]     at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
[2022-09-30T19:45:38.258Z]     at async pF (/home/mcartoixa/.vscode/extensions/ms-vscode-remote.remote-containers-0.251.0/dist/spec-node/devContainersSpecCLI.js:222:1353)
[2022-09-30T19:45:38.258Z]     at async dF (/home/mcartoixa/.vscode/extensions/ms-vscode-remote.remote-containers-0.251.0/dist/spec-node/devContainersSpecCLI.js:206:2361)
[2022-09-30T19:45:38.258Z]     at async DF (/home/mcartoixa/.vscode/extensions/ms-vscode-remote.remote-containers-0.251.0/dist/spec-node/devContainersSpecCLI.js:263:2177)
[2022-09-30T19:45:38.258Z]     at async to (/home/mcartoixa/.vscode/extensions/ms-vscode-remote.remote-containers-0.251.0/dist/spec-node/devContainersSpecCLI.js:263:3110)
[2022-09-30T19:45:38.259Z]     at async Ak (/home/mcartoixa/.vscode/extensions/ms-vscode-remote.remote-containers-0.251.0/dist/spec-node/devContainersSpecCLI.js:383:8108)
[2022-09-30T19:45:38.259Z]     at async Ok (/home/mcartoixa/.vscode/extensions/ms-vscode-remote.remote-containers-0.251.0/dist/spec-node/devContainersSpecCLI.js:383:7864)
[2022-09-30T19:45:38.261Z] Stop (699 ms): Run: /snap/code/108/usr/share/code/code --ms-enable-electron-run-as-node /home/mcartoixa/.vscode/extensions/ms-vscode-remote.remote-containers-0.251.0/dist/spec-node/devContainersSpecCLI.js up --user-data-folder /home/mcartoixa/.config/Code/User/globalStorage/ms-vscode-remote.remote-containers/data --workspace-folder /home/mcartoixa/Développement/air-indemnite.com/airindemnite_client --workspace-mount-consistency cached --id-label devcontainer.local_folder=/home/mcartoixa/Développement/air-indemnite.com/airindemnite_client --log-level debug --log-format json --config /home/mcartoixa/Développement/air-indemnite.com/airindemnite_client/.devcontainer/devcontainer.json --default-user-env-probe loginInteractiveShell --mount type=volume,source=vscode,target=/vscode,external=true --skip-post-create --update-remote-user-uid-default on --mount-workspace-git-root true
[2022-09-30T19:45:38.262Z] Exit code 1
```
The settings that will work on Ubuntu are:
```yml
    security_opt:
      - seccomp:.devcontainer/chrome.json
      - seccomp:.devcontainer/yarn.json
```
They will fail on Windows 10 with the following logs:
```
[2022-09-30T19:50:10.316Z] Writing docker-compose.devcontainer.containerFeatures-1664567410316.yml to d:\Users\Mathieu Cartoixa\AppData\Roaming\Code\User\globalStorage\ms-vscode-remote.remote-containers\data\docker-compose
[2022-09-30T19:50:10.317Z] Start: Run: docker-compose --project-name airindemnite_client_devcontainer -f d:\Users\Mathieu Cartoixa\Development\air-indemnite.com\airindemnite_client\.devcontainer\docker-compose.yml -f d:\Users\Mathieu Cartoixa\AppData\Roaming\Code\User\globalStorage\ms-vscode-remote.remote-containers\data\docker-compose\docker-compose.devcontainer.containerFeatures-1664567410316.yml up -d
[2022-09-30T19:50:10.465Z] 
[2022-09-30T19:50:10.687Z] 
[+] Running 3/0
 - Container airindemnite_client_devcontainer-glue-1                 Running 0.0
s
 - Container airindemnite_client_devcontainer-database-1             Running 0.0
s
 - Container airindemnite_client_devcontainer-cache-1                Running 0.0
s
 - Container airindemnite_client_devcontainer-airindemnite_client-1  Creating 0.
0s
opening seccomp profile (.devcontainer/chrome.json) failed: open d:\Users\Mathie
u Cartoixa\Development\air-indemnite.com\airindemnite_client\.devcontainer\.devc
ontainer\chrome.json: The system cannot find the path specified.
[2022-09-30T19:50:10.807Z] Stop (490 ms): Run: docker-compose --project-name airindemnite_client_devcontainer -f d:\Users\Mathieu Cartoixa\Development\air-indemnite.com\airindemnite_client\.devcontainer\docker-compose.yml -f d:\Users\Mathieu Cartoixa\AppData\Roaming\Code\User\globalStorage\ms-vscode-remote.remote-containers\data\docker-compose\docker-compose.devcontainer.containerFeatures-1664567410316.yml up -d
[2022-09-30T19:50:10.810Z] Error: Command failed: docker-compose --project-name airindemnite_client_devcontainer -f d:\Users\Mathieu Cartoixa\Development\air-indemnite.com\airindemnite_client\.devcontainer\docker-compose.yml -f d:\Users\Mathieu Cartoixa\AppData\Roaming\Code\User\globalStorage\ms-vscode-remote.remote-containers\data\docker-compose\docker-compose.devcontainer.containerFeatures-1664567410316.yml up -d
[2022-09-30T19:50:10.810Z]     at pF (d:\Users\Mathieu Cartoixa\.vscode\extensions\ms-vscode-remote.remote-containers-0.251.0\dist\spec-node\devContainersSpecCLI.js:222:2970)
[2022-09-30T19:50:10.810Z]     at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
[2022-09-30T19:50:10.810Z]     at async dF (d:\Users\Mathieu Cartoixa\.vscode\extensions\ms-vscode-remote.remote-containers-0.251.0\dist\spec-node\devContainersSpecCLI.js:206:2361)
[2022-09-30T19:50:10.810Z]     at async DF (d:\Users\Mathieu Cartoixa\.vscode\extensions\ms-vscode-remote.remote-containers-0.251.0\dist\spec-node\devContainersSpecCLI.js:263:2177)
[2022-09-30T19:50:10.811Z]     at async to (d:\Users\Mathieu Cartoixa\.vscode\extensions\ms-vscode-remote.remote-containers-0.251.0\dist\spec-node\devContainersSpecCLI.js:263:3110)
[2022-09-30T19:50:10.811Z]     at async Ak (d:\Users\Mathieu Cartoixa\.vscode\extensions\ms-vscode-remote.remote-containers-0.251.0\dist\spec-node\devContainersSpecCLI.js:383:8108)
[2022-09-30T19:50:10.811Z]     at async Ok (d:\Users\Mathieu Cartoixa\.vscode\extensions\ms-vscode-remote.remote-containers-0.251.0\dist\spec-node\devContainersSpecCLI.js:383:7864)
[2022-09-30T19:50:10.821Z] Stop (20776 ms): Run: D:\Program Files\Microsoft VS Code\Code.exe --ms-enable-electron-run-as-node d:\Users\Mathieu Cartoixa\.vscode\extensions\ms-vscode-remote.remote-containers-0.251.0\dist\spec-node\devContainersSpecCLI.js up --user-data-folder d:\Users\Mathieu Cartoixa\AppData\Roaming\Code\User\globalStorage\ms-vscode-remote.remote-containers\data --workspace-folder d:\Users\Mathieu Cartoixa\Development\air-indemnite.com\airindemnite_client --workspace-mount-consistency cached --id-label devcontainer.local_folder=d:\Users\Mathieu Cartoixa\Development\air-indemnite.com\airindemnite_client --log-level debug --log-format json --config d:\Users\Mathieu Cartoixa\Development\air-indemnite.com\airindemnite_client\.devcontainer\devcontainer.json --default-user-env-probe loginInteractiveShell --mount type=volume,source=vscode,target=/vscode,external=true --skip-post-create --update-remote-user-uid-default on --mount-workspace-git-root true
[2022-09-30T19:50:10.821Z] Exit code 1
```

**Comment by chrmarti** [comment]
Could you check if you are using the same versions of Docker Compose on your machines? I would expect the paths to always be relative to the docker-compose.yml's containing folder.

**Comment by mcartoixa** [comment]
Well indeed the version was wrong on Ubuntu! Thanks!

This was tested on 2 independent Ubuntu machines, which must have lived similar lives... Installation was done [using the docker repository](https://docs.docker.com/engine/install/ubuntu/#set-up-the-repository), some time ago now (it's _Docker Engine_, then). But `docker compose version` returned `docker-compose version 1.29.2, build unknown`, though version 2.10 was expected... `sudo apt install docker-compose-plugin` worked, but did not change a thing. Turns out there was a version of _Docker Compose_ living in `/usr/bin/` that preempted any other installation. It preempted _Docker Desktop_, which I had tried too...

Fix : whatever you're using (Engine or Desktop), make sure to `sudo apt remove docker-compose`...
